src/general/utils/ScreenUtils.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Godot;
+
+/// <summary>
+///   Utility for screen and view-related effects
+/// </summary>
+public static class ScreenUtils
+{
+    /// <summary>
+    ///   Applies barrel distortion to the given position
```

Comment:
I'd like to see a remarks section added here saying that this must match the math in `Chromatic.gdshader`.

Also I'd like to see a comment in `Chromatic.gdshader` saying that if the math is updated then the math in this file needs to be updated.

Reply:
Added the comments now